tests/ci/create-osbuild.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -euo pipefail
+
+# following steps as noted on https://hackmd.io/ZbUOkeIXTjaP7XVpTSsrNw?view
+
+# Get OS data.
+source /etc/os-release
+
+ID=${ID:-}
+ARCH=$(arch)
+OS_VARIANT=${OS_VARIANT:-}
+IMAGE_TYPE=${IMAGE_TYPE:-}
+UUID=${UUID:-local}
+BOOT_LOCATION="http://mirror.centos.org/centos/8-stream/BaseOS/${ARCH}/os/"
```

Comment:
I think we don't need these variables:
- OS_VARIANT
- IMAGE_TYPE
- BOOT_LOCATION

But maybe we could use something like:
```
DISK_FILE="image_output/image/disk.img"
IMAGE_FILE="osbuild-${ARCH}-${UUID}.img"
```
The first is the path of the generated file and the second is a composed name to rename it, so we can upload it later to S3 with unique name.